### PR TITLE
feat: delete conversations notification (AR-2039)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2039" title="AR-2039" target="_blank">StoryAR-2039</a>  Admin deletes a group conversation that I belong to
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

This PR adds the notification handling when a group-conversation is deleted by it's creator.

Needs releases with: https://github.com/wireapp/kalium/pull/763

- [x] GitHub link to other pull request 

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

### Attachments (Optional)

https://user-images.githubusercontent.com/5806454/183061379-7c0c3423-4240-492b-8d69-9a356674860a.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by ohassine — APPROVED** [review]
lgtm 👌

File: .github/workflows/gradle-run-unit-tests.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ on:
         - 'develop'
   workflow_call:
 
+concurrency:
+    group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+    cancel-in-progress: true
```
Comment (gongracr) [inline .github/workflows/gradle-run-unit-tests.yml:13]: What's this for? to cancel an ongoing GH action if a new commit arrives?

**Review by gongracr — APPROVED** [review]

File: .github/workflows/gradle-run-unit-tests.yml (hunk 1 above)
Comment (yamilmedina) [reply .github/workflows/gradle-run-unit-tests.yml:13]: oh yeah, the same filter we have on kalium, to save some 💸

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-834/7/) **failed**.

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-834/9/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4107264)(4107264)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4107264)(4107264)-dev-release.apk)

**Pull request merged**